conf/messages.en

```diff
@@ -456,4 +456,6 @@ group.member.already.registered.title = You cannot add {0} to the group
 group.member.already.registered.detail1 = {0} has already registered for PPT, so cannot be added to the group.
 group.member.already.registered.detail2 = {0} will need to de-register for PPT before you can add them to the group.
 
-amendRegistration.organisationDetails.title = Your PPT registration
\ No newline at end of file
+amendRegistration.organisationDetails.title = Your PPT registration
```

Comment:
Is this from Ryan's PR already on main?

Reply:
It's just a newline at the end of a line which was added by my PR. No probs.